src/fetchtastic/download/cli_integration.py

```
@@ -418,19 +432,14 @@ def log_download_results_summary(
-            log.info(f"Latest APK: {latest_apk_version}")
-        if latest_apk_prerelease:
-            log.info(f"Latest APK prerelease: {latest_apk_prerelease}")
-        else:
-            log.info("Latest APK prerelease: none")
-
-        if latest_desktop_version:
-            log.info(f"Latest desktop: {latest_desktop_version}")
-        if latest_desktop_prerelease:
-            log.info(f"Latest desktop prerelease: {latest_desktop_prerelease}")
+        if latest_client_app:
+            log.info(f"Latest Meshtastic Client release: {latest_client_app}")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)